client/public/employeePage/employee.js

```diff
@@ -0,0 +1,90 @@
+const addHomeAddressForm = document.getElementById("addHomeAddressForm");
+const vacationRequestForm = document.getElementById("vacationRequestForm")
+let tbody=document.getElementById("tbody");
+let messages=document.getElementById("messages");
+
+Date.prototype.toDateInputValue = (function() {
+    let local = new Date(this);
+    local.setMinutes(this.getMinutes() - this.getTimezoneOffset());
+    return local.toJSON().slice(0,10);
+});
+
+document.getElementById('vacationRequestFrom').value = new Date().toDateInputValue();
+document.getElementById('vacationRequestTo').value = new Date().toDateInputValue();
+
+//fetch user function
+const display = async() => {
+    try
+    {
+        const response = await fetch("http://localhost:4000/api/employeeInfo?id=1", {method: "GET"})
+        return response.json();
+    }
+    catch(error)
+    {
+        throw error;
+    }
+}
+
+//display user function
+window.onload = async function employeeDisplay () {
+    const displayVar = await display();
+    console.log(displayVar);
+    // let users = displayVar["users"];
+    let td = document.createElement('tr')
+        td.innerHTML = `
+                        <td>email</td>
+                        <td>${displayVar}</td>
+                        <td>role</td>
+                        <td>vacation days</td>`
+        tbody.append(td);
+    return td;
+}
+
+//fetch home address function
+async function homeAddress(homeAddressVar) {
+    try
+    {
+        const response = await fetch(`http://localhost:4000/api/home_address?id=1&home_address=${homeAddressVar}`,{method:"put"})
+        return response.json();
+    }
+    catch(error)
+    {
+        throw error;
```

Comment:
same here. if there is an error, something should happen in the ui to indicate it.